tools/setup_perf_cluster.sh

```diff
@@ -113,6 +128,12 @@ function install_istio() {
   Execute sh -c 'sed -e "s/_debug//g" install/kubernetes/istio-auth.yaml | egrep -v -e "- (-v|\"2\")" | kubectl apply -f -'
 }
 
+# assumes run from istio/ (or release) directory
+function delete_istio() {
+  # Use the non debug ingress and remove the -v "2"
+  Execute sh -c 'sed -e "s/_debug//g" install/kubernetes/istio-auth.yaml | egrep -v -e "- (-v|\"2\")" | kubectl delete -f -'
```

Comment:
I think 
```
kubectl delete -f install/kubernetes/istio-auth.yaml
```
would work the same as the image and flags don't matter during delete